index.bs

```diff
@@ -827,6 +835,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 :   <code><dfn for="credentialCreationData">clientDataJSONResult</dfn></code>
                 ::  whose value is the bytes of |clientDataJSON|.
 
+                :   <code><dfn for="credentialCreationData">AttestationConveyancePreferenceOption</dfn></code>
```

Comment:
It's in a struct, so maybe it doesn't matter, but we usually have field names start with lower-case.

Reply:
ok.